version.sbt

```diff
@@ -0,0 +1 @@
+ThisBuild / version := "0.41.0-SNAPSHOT"
```

Comment:
TODO: can we publish snapshots using the reusable workflow or should we instead be using `publishLocal`? Will also need to update the readme with new instructions about how to release

Reply:
The issue https://github.com/guardian/gha-scala-library-release-workflow/issues/10 tracks this - currently, there's no support for `SNAPSHOT` releases with `gha-scala-library-release-workflow`, but I do have some ideas about how to make arbitrary pre-release versions from branches that I hope to implement in the workflow soon.

`publishLocal` works well if you are building everything on your laptop, unfortunately it's not very helpful if you want to try out a pre-release of this library in the CI build of another project that consumes this library - the CI build won't have access to your personal local artifacts.